### PR TITLE
910330-Revamped the UG contents in SfComboBox and SfCarousel controls.

### PR DESCRIPTION
**Pull request by SivaranjithN**
Hi Team,
I have revamped the UG contents in .NET MAUI SfComboBox and SfCarousel controls.

Regards,
Sivaranjith N.

### CODE REVIEW

File: MAUI/Carousel-View/getting-started.md [hunk 1]
```diff
@@ -186,9 +186,22 @@ The following code example illustrates how to add the collection in Carousel,
         <local:CarouselViewModel/>
     </ContentPage.BindingContext>
 
-    <carousel:SfCarousel x:Name="carousel"
-                         ItemsSource="{Binding CarouselItems}"/>
-</ContentPage>
+    <ContentPage.Resources>
```
Comment (PrithisIyyappan) [inline MAUI/Carousel-View/getting-started.md:189]: Remove height and width request.

File: MAUI/Carousel-View/getting-started.md [hunk 2]
```diff
@@ -229,7 +256,7 @@ namespace CarouselSample
 {% highlight xaml %}
 
 <carousel:SfCarousel x:Name="carousel"
-                     ItemsSource="{Binding CarouselItems}"
+                     ItemsSource="{Binding ImageCollection}"
```
Comment (PrithisIyyappan) [inline MAUI/Carousel-View/getting-started.md:259]: Include item template

File: MAUI/Carousel-View/getting-started.md [hunk 3]
```diff
@@ -258,7 +285,7 @@ We can bring particular item to the center of the screen using [SelectedIndex](h
...
```
Comment (PrithisIyyappan) [inline MAUI/Carousel-View/getting-started.md:288]: Modify item source in C# too.

File: MAUI/Carousel-View/getting-started.md (hunk 1 above)
Comment (SivaranjithN) [reply MAUI/Carousel-View/getting-started.md:189]: Addressed.

File: MAUI/Carousel-View/getting-started.md (hunk 3 above)
Comment (SivaranjithN) [reply MAUI/Carousel-View/getting-started.md:289]: Addressed.

File: MAUI/Carousel-View/getting-started.md [hunk 4]
```diff
@@ -204,12 +215,26 @@ namespace CarouselSample
...
             InitializeComponent();
-
             CarouselViewModel carouselViewModel = new CarouselViewModel();
-            this.BindingContext = carouselViewModel;
 
-            SfCarousel carousel = new SfCarousel();
-            carousel.SetBinding(SfCarousel.ItemsSourceProperty, "CarouselItems");
+            SfCarousel carousel = new SfCarousel()
+            {
+                HeightRequest = 400,
```
Comment (PrithisIyyappan) [inline MAUI/Carousel-View/getting-started.md:222]: Check this.

Comment (SivaranjithN) [reply MAUI/Carousel-View/getting-started.md]: Addressed.

**Review by Choza-rajan — APPROVED** [review]

**Pull request merged**